commons/src/main/java/fr/free/nrw/commons/upload/ShareActivity.java

```diff
@@ -170,7 +180,16 @@ public void onCreate(Bundle savedInstanceState) {
             mimeType = intent.getType();
         }
 
-        ImageLoader.getInstance().displayImage(mediaUri.toString(), backgroundImageView);
+        mediaUriString = mediaUri.toString();
+        Log.d("Image", "Uri: " + mediaUriString);
+
+        ImageProcessing imageObj = new ImageProcessing(this, mediaUri);
```

Comment:
Since ImageProcessing could be misconstrued with filters, face detection, or a whole bunch of other things, maybe this should be called something else? ImageMetaDataProcessor? GpsExtractor? idk